ipv6/generate-ipv6-address/Makefile

```diff
@@ -0,0 +1,37 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=generate-ipv6-address
+PKG_VERSION:=0.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=generate-ipv6-address-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://www.irif.fr/~jch/software/files/
+PKG_HASH:=e1356d245d5f891fa39b796a8a2deefdaa89f08130dc97a378738ca5ed9a20fa
+PKG_INSTALL:=1
+
+PKG_MAINTAINER:=Nick Hainke <vincent@systemli.org>
```

Comment:
no license?

Reply:
I searched now for the header text in a c file and I think it is MIT. I will ad this. but what is then the `PKG_LICENSE_FILES`?